metric-anomaly-data-model/src/main/proto/org/hypertrace/alert/engine/eventcondition/config/service/v1/metric_anomaly_event_condition.proto

```diff
@@ -87,6 +89,10 @@ message StaticThresholdCondition {
   Severity severity = 4;
 }
 
+message DynamicThresholdCondition {
```

Comment:
I know everything is one single PR. But, one simple alert rule with an example just with this definition modification would have help. And breaking it into smaller PRs also helps in reviewing.

Reply:
yes I am working on tes

Reply:
Suggestion (might help) :
Can we review commit by commit instead of all the commits once? All similar changes can be grouped together into one commit with appropriate commit message?